packages/uui-combobox-list/lib/uui-combobox-list.element.ts

```diff
@@ -239,6 +239,16 @@ export class UUIComboboxListElement extends LitElement {
         break;
       }
 
+      //Space key
+      case ' ': {
```

Comment:
Do you think we could find some other way to recognize a space than a literal _whitespace_? 

It would seem from [this list](https://www.freecodecamp.org/news/javascript-keycode-list-keypress-event-key-codes/) that using the `event.code` instead of `event.key` gives you the option to detect the space key with `'Space'` instead of `' '`.